### ISSUE TITLE
breakhit are incorrect?

### ISSUE THREAD

**Issue by NotDark** [GrowtopiaNoobs/Growtopia_ItemsDecoder#12]
![image](https://user-images.githubusercontent.com/44568476/71590488-a51b2e00-2b63-11ea-8936-a2b4fb87958f.png)

breakhit for dirt should be 3 
![image](https://user-images.githubusercontent.com/44568476/71590511-bcf2b200-2b63-11ea-8e4e-05a37c97da9e.png)

**Comment by NotDark** [comment]
nvm i found it
you need divide by 6 

18/6 = 3

**Comment by ardakurucim** [comment]
there are 2 dirts idk why

**Comment by MarmoriPelaa** [comment]
Yes, you are correct. Just change the https://github.com/GrowtopiaNoobs/Growtopia_ItemsDecoder/blob/3030090e23aa1829eb44e7ac37f37658a62100cb/items_decoder.cpp#L66
to
```c++
unsigned char breakHits = 0;
```
